stellargraph/datasets/datasets.py

```diff
@@ -803,3 +803,54 @@ def load(self):
         )
 
         return StellarGraph(nodes=nodes, edges=edges, edge_weight_column="time"), edges
+
+
+class METR_LA(
+    DatasetLoader,
+    name="METR-LA",
+    directory_name="METR-LA",
+    url="https://github.com/lehaifeng/T-GCN/tree/master/data",
+    url_archive_format="n/a",
+    expected_files=["los_speed.csv", "los_adj.csv"],
+    description="This traffic dataset contains traffic information collected from loop detectors in the highway of Los Angeles County (Jagadish et al., 2014).",
+    source="https://github.com/lehaifeng/T-GCN/tree/master/data",
+):
+    def load(self):
+        los_adj = pd.read_csv(r"data/los_adj.csv", header=None)
```

Comment:
To get this in a similar format to the rest of our dataset this should call `self.download()` and then load the dataset from `~/stellargraph-datasets`. Something like:

```
def load(self):
    self.download()
    paths = [self._resolve_path(f) for f in expected_files]
    los_adj = pd.read_csv(paths[1], header=None)
    adj = np.mat(los_adj)
    los_tf = pd.read_csv(paths[0])
```

Reply:
I think `download()` won't do the right thing here, because `url` needs to point to a single file that contains the data (e.g. a zip file, or similar). Getting an appropriate zip file out of a GitHub repo is a bit fiddly: we've taken to just uploading it to figshare.com separately.

Given all that, and in the interests of getting this landed sooner rather than later, I think it would be appropriate to leave this class as is, but call it experimental:

```python
@experimental(reason="the data isn't downloaded automatically", issues=[9999])
class METR_LA(
```

where a follow-up issue is filed about this, and `9999` is replaced with its number.

(It's great to this parsing code all packaged up!)